cpp/CMakeLists.txt

```diff
@@ -361,44 +371,63 @@ if(Doxygen_FOUND)
 endif()
 
 
-add_executable(cuopt_cli cuopt_cli.cpp)
-target_compile_options(cuopt_cli
-  PRIVATE "$<$<COMPILE_LANGUAGE:CXX>:${CUOPT_CXX_FLAGS}>"
-  "$<$<COMPILE_LANGUAGE:CUDA>:${CUOPT_CUDA_FLAGS}>"
-)
+if(NOT BUILD_LP_ONLY)
+  add_executable(cuopt_cli cuopt_cli.cpp)
+  target_compile_options(cuopt_cli
+    PRIVATE "$<$<COMPILE_LANGUAGE:CXX>:${CUOPT_CXX_FLAGS}>"
+    "$<$<COMPILE_LANGUAGE:CUDA>:${CUOPT_CUDA_FLAGS}>"
+  )
 
-target_include_directories(cuopt_cli
-  PRIVATE
-  "${CMAKE_CURRENT_SOURCE_DIR}/src"
-  PUBLIC
-  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-  "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>"
-  "$<INSTALL_INTERFACE:include>"
-)
+  target_include_directories(cuopt_cli
+    PRIVATE
+    "${CMAKE_CURRENT_SOURCE_DIR}/src"
+    PUBLIC
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>"
+    "$<INSTALL_INTERFACE:include>"
+  )
 
-target_link_libraries(cuopt_cli
-  PUBLIC
-  cuopt
-  OpenMP::OpenMP_CXX
-  PRIVATE
-  papilo-core
-)
-set_property(TARGET cuopt_cli PROPERTY INSTALL_RPATH "$ORIGIN/../${lib_dir}")
+  target_link_libraries(cuopt_cli
+    PUBLIC
+    cuopt
+    OpenMP::OpenMP_CXX
+    PRIVATE
+    papilo-core
+  )
+  set_property(TARGET cuopt_cli PROPERTY INSTALL_RPATH "$ORIGIN/../${lib_dir}")
 
-# adds the cuopt_cli executable to the runtime deb package
-install(TARGETS cuopt_cli
-  COMPONENT runtime
-  RUNTIME DESTINATION ${_BIN_DEST}
-)
+  # adds the cuopt_cli executable to the runtime deb package
+  install(TARGETS cuopt_cli
+    COMPONENT runtime
+    RUNTIME DESTINATION ${_BIN_DEST}
+  )
+endif()
+
+
+option(BUILD_MIP_BENCHMARKS "Build MIP benchmarks" OFF)
+if(BUILD_MIP_BENCHMARKS AND NOT BUILD_LP_ONLY)
+  add_executable(solve_MIP ../benchmarks/linear_programming/cuopt/run_mip.cpp)
+  target_compile_options(solve_MIP
+    PRIVATE "$<$<COMPILE_LANGUAGE:CXX>:${CUOPT_CXX_FLAGS}>"
+    "$<$<COMPILE_LANGUAGE:CUDA>:${CUOPT_CUDA_FLAGS}>"
+  )
+  target_link_libraries(solve_MIP
+    PUBLIC
+    cuopt
+    OpenMP::OpenMP_CXX
+    PRIVATE
+    papilo-core
+  )
+endif()
 
-option(BUILD_BENCHMARKS "Build benchmarks" ON)
-if(BUILD_BENCHMARKS)
-  add_executable(solve_MPS_file ../benchmarks/linear_programming/cuopt/run_mip.cpp)
-  target_compile_options(solve_MPS_file
+option(BUILD_LP_BENCHMARKS "Build LP benchmarks" OFF)
+if(BUILD_LP_BENCHMARKS)
+  add_executable(solve_LP ../benchmarks/linear_programming/cuopt/run_pdlp.cu)
```

Comment:
For benchmarking, I think we should directly use the cuopt_cli, because that's what everyone will use